info.md

```diff
@@ -20,20 +20,44 @@ Configuration variables:
   - **dewpoint**: Dewpoint (°C or °F)
   - **heat_index**: Heat index (°C or °F)
   - **temp**: Temperature (°C or °F)
+  - **temp_indoor**: Temperature (°C or °F)
+  - **temp_day_max**: Today MAX temperature (°C or °F)
+  - **temp_day_min**: Today MIN temperature (°C or °F)
   - **humidex**: Humidex (°C or °F)
   - **wind_degrees**: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise.
   - **wind_dir**: Wind Direction as string ie.: N, NW, etc.
   - **wind_gust**: Wind Gust (km/h or mph)
   - **wind_speed**: Wind Speed (km/h or mph)
   - **symbol**: Symbol
   - **daily_rain**: Daily Rain (mm or in)
+  - **monthly_rain**: Daily Rain (mm or in)
+  - **daily_rain**: Daily Rain (mm or in)
+  - **yearly_rain**: Daily Rain (mm or in)
+  - **rain_rate**: Daily Rain (mm or in)
   - **pressure**: Pressure (hPa or inHg)
   - **humidity**: Relative humidity (%)
+  - **humidity_indoor**: Relative humidity (%)
   - **cloud_height**: Cloud Height (m or ft)
   - **forecast**: string based output ie.: night showers
 
-A full configuration example can be found below:
+## Install via [HACS](https://github.com/custom-components/hacs)
```

Comment:
there is no need to have an install info in HACS store, there is already a button to install